cpp/src/community/triangles_counting.cu

```diff
@@ -16,17 +16,19 @@
 
 #include <cuda_runtime.h>
 
+#include <raft/cudart_utils.h>
 #include <algorithms.hpp>
 #include <graph.hpp>
 
-#include <nvgraph/include/sm_utils.h>
-#include <nvgraph/include/nvgraph_error.hxx>
+#include <utilities/sm_utils.h>
+#include <utilities/error.hpp>
 
 #include <rmm/thrust_rmm_allocator.h>
 #include <rmm/device_buffer.hpp>
 
 #include <thrust/iterator/counting_iterator.h>
 
+#include <raft/utils/sm_utils.hpp>
```

Comment:
We're including utilities/sm_utils.h in https://github.com/rapidsai/cugraph/pull/980/files#diff-a8b84d43fbea60927dbb836c2f31b577R23

and raft/utils/sm_utils.hpp here.

What are the differences between the two? Do we need both? Can't we consolidate the two to one?

Reply:
The `raft` one is much more comprehensive. The `cugraph` one is much shorter and provides slightly different utils.